config/initializers/gem-plugin_config/sanitizer_config.rb

```diff
@@ -36,10 +36,6 @@ module Config
         'img' => {'src' => ['http', 'https', :relative]},
         'q' => {'cite' => ['http', 'https', :relative]}
       },
-      
-      # TODO: This can be removed once we upgrade sanitizer gem, AO3-5801
-      # I would leave the tests we added in AO3-5974 though.
-      remove_contents: %w[iframe math noembed noframes noscript plaintext script style svg xmp]
```

Comment:
I believe this issue was resolved earlier than my PR, when `sanitize` was updated to 6.0.0 in AO3-6085 (Upgrade to Rails 6). Should we update Gemfile as well? In Gemfile `sanitize` is still '>= 4.6.5'.

Reply:
Now ParagraphMaker (in a2dafbcc43aadeeed377ea7e4365883f272688b6) uses this. In July I thought it was unnecessary, but I'm now less confident. If I recall it correctly, it wasn't exactly related to HTML5, but it was merely something I found while working on the HTML5 issue. I'll revert it for now.